server/server.go

```diff
@@ -236,6 +237,10 @@ func NewServer(logger *zap.Logger, storeInstance *store.Store, loggerLevel *zap.
 	s.registerLabelRoutes(apiGroup)
 	s.registerSubscriptionRoutes(apiGroup)
 	s.registerSheetRoutes(apiGroup)
+	// Register healthz endpoint.
```

Comment:
not sure if healthz is a common convention outside of google.

Reply:
I don't think there's a defined pattern as long as we stick to it. sourcegraph/render uses healthz. I'm also okay with health or status, but I think the suffix "z" is more of a telemetry stream pattern.

https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/cmd/symbols/internal/api/handler.go?L24:35&subtree=true
https://render.com/docs/deploys#health-checks
https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-6.0

Reply:
Oh, looks like it's a convention now https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f

Reply:
Thanks for the pointer! Yes, I think everyone follows Google pattern regardless of existing usage - greenfield.